pom.xml

```diff
@@ -75,6 +75,16 @@
         <argLine></argLine>
     </properties>
 
+    <repositories>
```

Comment:
I don't think we want to add liquibase-pro's repo to the public source's pom? That is something we can uniquely do and so we shouldn't be setting everyone in the world to be using it by default. Instead add it to the settings.xml on the build server. 

Also on there we probably want/need a different id than `github` since that will maybe be expected to be the current projects github package repo vs. a one for a different dependency?